### ISSUE TITLE
Detect popup blocker ? On open workspace

### ISSUE THREAD

**Issue by hexa00** [theia-ide/theia#662]
Once you've opened a workspace, selecting File->Open on a directory has no effect.

**Comment by marcdumais-work** [comment]
I think it was mentioned that we're supposed to refresh the browser for the change to take effect?

**Comment by hexa00** [comment]
True that works.

But the user should not have to do that.

**Comment by akosyakov** [comment]
I've just tried on the master:
- `yarn start .` from the browser directory
- `File` -> `Open..`, open the electron directory
- a new window is opened with the electron directory as the workspace root, the old window is preserved with the browser directory as the workspace root

What do you do?

**Comment by hexa00** [comment]
I just noticed that chrome blocks this as a popup!

So that's why.

The popup block notification isn't very visible so I had not noticed before...

I wonder if we could detect that?

**Comment by lmcbout** [comment]
Another thing, when a workspace is opened, select File-> Open... allows you to open a parent of the current workspace, no other workspace are available to be opened, so we cannot select what I am looking for.

**Comment by lmcbout** [comment]
To explain my previous comment,
Check the video. 
1- select a sub-sub folder as a workspace
2- Use open again to select another workspace: the parent folder is selected , it should show the current workspace and the user needs to select in the parent folder on top of it, not inside the tree list
![peek 2017-10-19 15-07](https://user-images.githubusercontent.com/419051/31789424-145cc2bc-b4e0-11e7-858e-0e1b35ea34d2.gif)

**Comment by hexa00** [comment]
@lmcbout please open another issue about your last comment.

**Comment by lmcbout** [comment]
I removed the comments and create another issue: 691

**Comment by hexa00** [comment]
Changed the title a bit

**Comment by kittaakos** [comment]
I need to verify it after #838.

**Comment by kittaakos** [comment]
It does not happen anymore. I am closing this.
